File: electron/services/admins.service.ts
import { BaseRepository } from '../db/base.repository';
import { BaseService } from './base.service';
import type { Admin } from '../models';

const adminRepo = new BaseRepository<Admin>('Admins', 'AdminID');

export class AdminsService extends BaseService<Admin> {
  constructor() {
    super(adminRepo);
  }
}

export const adminsService = new AdminsService();

File: reference/old/Ceremony.Service/AdminsService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using Ceremony.Models;

namespace Ceremony.Service
{
    public class AdminsService : BaseService<Admins>
    {
        public AdminsService()
        {
            repository = new GenericRepository<Admins>();
        }

        public AdminsService(CeremonyEntities context)
        {
            repository = new GenericRepository<Admins>(context);
        }
    }
}
